help/include/inbox.md

```diff
@@ -0,0 +1,41 @@
+The **Inbox** is the default view in the Zulip mobile app. It's a great way to
+get an overview of all the unmuted [conversations](/help/recent-conversations)
+where you have unread messages, excluding [inactive
+streams](/help/manage-inactive-streams).
+
+The **Inbox** view shows your unread direct message conversations, followed by
+all topics with unread messages, grouped by stream. The list of streams is
+sorted alphabetically, with [pinned streams](/help/pin-a-stream) at the top.
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{relative|message|inbox}
+
+1. The filters at the top help you quickly find relevant conversations.
+   You can click the icon to the left of a stream name to **collapse**
+   (<i class="zulip-icon zulip-icon-arrow-down"></i>) the list of topics in the
```

Comment:
How about making this instruction something like this?

> The filters at the top help you quickly find relevant conversations. You can filter by stream or conversation.

And then the collapse/uncollapse instruction can be a tip, like on the mobile instructions (maybe even with the exact same wording, modulo the icons)?